src/modules/blocks/ticket/container-header/price/template.js

```diff
@@ -29,7 +29,14 @@ const TicketContainerHeaderPriceLabel = ( {
 	const getAvailableLabel = () => (
 		isUnlimited
 			? __( 'unlimited', 'event-tickets' )
-			: `${ available } ${ __( 'available', 'event-tickets' ) }`
+			: (
+				<>
+					<span className="tribe-editor__ticket__container-header-label__available">
+						{ available }
+					</span>
+					{ __( 'available', 'event-tickets' ) }
```

Comment:
We may want to use the `i18n` package from WP to make this translatable: https://developer.wordpress.org/block-editor/reference-guides/packages/packages-i18n/#usage

In some languages the format could be `X available`, but in some others, the flow is `word number`